bot.user.js

```diff
@@ -721,3 +856,54 @@ window.initBot();
             }
         };
         */
+		
+					//collsionPoints = window.getCollisionPoints();
+			//var center = [window.mc.width / 2, window.mc.height / 2];
+			//
+            //lineStart = window.mapToMouse(window.snake.xx, window.snake.yy);
+			//lineStart = window.mouseToScreen(lineStart[0], lineStart[1]);
+            //lineStart = window.screenToCanvas(lineStart[0], lineStart[1]);
+			//
+			//for (points in collsionPoints){
+			//	circle = {x: collsionPoints[points].x + collsionPoints[points].fx,
+			//			  y: collsionPoints[points].y + collsionPoints[points].fy,
+			//			  radius: 22 * collsionPoints[points].sc * window.getScale()};
+			//			  
+			//	interceptOnCircle({x: lineStart[0], y: lineStart[1]},{x: drawGoalCoordinates[0], y: drawGoalCoordinates[1]}, collisionScreenToCanvas(circle));
+			//}
+//        }
+//    }
+//};
+
+//line intercept
+/*function interceptOnCircle(p1,p2,c){
+    //p1 is the first line point
+    //p2 is the second line point
+    //c is the circle's center
+    //r is the circle's radius
+	
+	//window.drawDot(c.x, c.y, c.radius, 'red', true);
+    var p3 = {x:p1.x - c.x, y:p1.y - c.y} //shifted line points
+    var p4 = {x:p2.x - c.x, y:p2.y - c.y}
+
+    var m = (p4.y - p3.y) / (p4.x - p3.x); //slope of the line
+    var b = p3.y - m * p3.x; //y-intercept of line
+
+    var underRadical = Math.pow((Math.pow(c.radius,2)*(Math.pow(m,2)+1)) ,2) -Math.pow(b,2); //the value under the square root sign 
+	
+	//var context = window.mc.getContext('2d');
+    //context.lineWidth = 5 * window.getScale();
+    //context.strokeStyle = '#00FF00' 
+    //context.moveTo(p1.x, p1.y);
+    //context.lineTo(p2.x, p2.y);
+    //context.stroke();
+	
+    if (underRadical < 0){
+    //line completely missed
+		console.log("MISS");
+        return false;
+    } else {
+		console.log("HIT");
+        return true;
+    }
+};*/
```

Comment:
What's your plan for these large chunks of commented code? Shouldn't they be used, or deleted?